modules/beopBidAdapter.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+**Module Name** : BeOp Bidder Adapter  
+**Module Type** : Bidder Adapter  
+**Maintainer** : tech@beop.io
+
+# Description
+
+Module that connects to BeOp's demand sources
+
+# Test Parameters
+```
+    var adUnits = [
+        {
+            code: 'in-article',
+            mediaTypes: {
+                banner: {
+                    sizes: [[300, 250]],  // a display size
+                }
+            },
+            bids: [
+                {
+                    bidder: "example",
```

Comment:
This should be your bidder code. Same change should be made on 2nd example ad unit.